src/components/EventSearchInput/EventSearchInput.js

```diff
@@ -12,6 +12,21 @@ class EventSearchInput extends Component {
 
     return (
       <div className={styles.inputSearchWrapper}>
+        <svg
```

Comment:
For the sake of consistency, it may be better to use the [icon set](https://gorangajic.github.io/react-icons/fa.html) we're already using in other places in the app, the FaSearch component in this case.  If you search for `<FaTwitter size={25} />` you'll find an example of where it's already being used.